### DISCUSSION TITLE
v4.1

### DISCUSSION THREAD

**Discussion by port19x** [pystardust/ani-cli#1038] [General] [answered]
## What's Changed
- Documentation
  - wget2 info for debian package by @Wiener234 in #998
- Bugfixes
  - aria2 continue interrupted downloads by @candrapersada in #1008
  - Http agent from linux to windows for privacy (and shuf removal) by @x746f6f72 in #1012
  - Ani-cli ascending episode sort by @athuld in #1019
  - set executable bit again by @port19x in #1021
  - previous,next,replay bug by @RaynardGerraldo in #1028
  - switch to curl by @71zenith in #1037
- Features
  - Change quality interactively #1006 by @RaynardGerraldo in #1016

## New Contributors
* @candrapersada made their first contribution in https://github.com/pystardust/ani-cli/pull/1008
* @x746f6f72 made their first contribution in https://github.com/pystardust/ani-cli/pull/1012
* @athuld made their first contribution in https://github.com/pystardust/ani-cli/pull/1019

**Full Changelog**: https://github.com/pystardust/ani-cli/compare/v4.0...v4.1

<hr /><em>This discussion was created from the release <a href='https://github.com/pystardust/ani-cli/releases/tag/v4.1'>v4.1</a>.</em>

**Comment by Elsie19** [comment, marked as answer]
Updated Pacstall in https://github.com/pacstall/pacstall-programs/pull/2481.
